src/components/Drawer/index.tsx

```diff
@@ -16,15 +17,17 @@ export default function Drawer({ isOpen, onClose, children, title, backgroundCol
   return (
     <DrawerRoot isOpen={isOpen} backgroundColor={backgroundColor}>
       {title && (
-        <Box display="flex" justifyContent="space-between" alignItems="center" padding="10px">
-          <Text color="text1" fontSize={24}>
+        <Box display="flex" justifyContent="space-between" alignItems="center" padding="20px">
+          <Text color="text1" fontSize={21} fontWeight={800}>
             {title}
           </Text>
         </Box>
       )}
 
-      <Box position="absolute" right={10} top={10}>
-        <CloseIcon onClick={onClose} color={theme.text4} />
+      <Box position="absolute" right={20} top={20}>
+        <CloseCircle onClick={onClose}>
+          <X color={theme.primary} size={10} />
+        </CloseCircle>
```

Comment:
Close icon is not in center align with title. Please fix right & top position.

![Screenshot 2022-09-05 at 3 35 54 PM](https://user-images.githubusercontent.com/6604146/188424651-819c56f2-0454-45c9-82fa-56be1fbf550c.png)

Reply:
This should be fine now, I fixed the conflicts.